src/silo/query_engine/filter_expressions/aa_symbol_equals.cpp

```diff
@@ -0,0 +1,92 @@
+#include "silo/query_engine/filter_expressions/aa_symbol_equals.h"
+
+#include <nlohmann/json.hpp>
+#include <vector>
+
+#include "silo/common/aa_symbols.h"
+#include "silo/database.h"
+#include "silo/query_engine/filter_expressions/or.h"
+#include "silo/query_engine/operators/bitmap_selection.h"
+#include "silo/query_engine/operators/complement.h"
+#include "silo/query_engine/operators/index_scan.h"
+#include "silo/query_engine/query_parse_exception.h"
+#include "silo/storage/database_partition.h"
+
+namespace silo::query_engine::filter_expressions {
+
+AASymbolEquals::AASymbolEquals(std::string aa_sequence_name, uint32_t position, char value)
+    : aa_sequence_name(std::move(aa_sequence_name)),
+      position(position),
+      value(value) {}
+
+std::string AASymbolEquals::toString(const silo::Database& /*database*/) const {
+   std::string res = std::to_string(position + 1) + std::to_string(value);
+   return res;
```

Comment:
```suggestion
   return std::to_string(position + 1) + std::to_string(value);
```
And maybe include the `aa_sequence_name`?